python/paddle/fluid/tests/unittests/test_activation_op.py

```diff
@@ -895,6 +896,58 @@ def test_check_grad(self):
         self.check_grad(['X'], 'Out')
 
 
+class TestTan(TestActivation):
+    def setUp(self):
+        np.random.seed(1024)
+        self.op_type = "tan"
+        self.init_dtype()
+        self.dtype = 'float32'
+        self.x_np = np.random.uniform(-1, 1, [10, 12]).astype(self.dtype)
+        self.place = paddle.CUDAPlace(0) if core.is_compiled_with_cuda() \
+            else paddle.CPUPlace()
+
+        out = np.tan(self.x_np)
+
+        self.inputs = {'X': OpTest.np_dtype_to_fluid_dtype(self.x_np)}
+        self.outputs = {'Out': out}
+
+    def test_check_grad(self):
+        if self.dtype == np.float16:
```

Comment:
From line 904, I think  you are using string 'float32' for self.dtype, but here you use "np.float16", should we change them to same type?

Reply:
this one is a skip condition, thx done